ui/apps/platform/src/Containers/NetworkGraph/external/ExternalIpsTable.tsx

```diff
@@ -101,7 +111,15 @@ function ExternalIpsTable({ scopeHierarchy }: ExternalIpsTableProps) {
                                 {data.map(({ entity, flowsCount }) => {
                                     return (
                                         <Tr key={entity.id}>
-                                            <Td dataLabel="Entity">{entity.externalSource.name}</Td>
+                                            <Td dataLabel="Entity">
+                                                <Button
+                                                    variant="link"
+                                                    isInline
+                                                    onClick={() => setSelectedEntity(entity)}
```

Comment:
I'm wondering what we would do in case you're looking at [this view](https://www.figma.com/proto/wWkk0s0ri1cuesnuEVsHDV/Network-IP?page-id=267%3A111131&node-id=2120-35326&node-type=frame&viewport=10521%2C-12321%2C0.32&t=7WyEZbFHlHxHGfEV-1&scaling=min-zoom&content-scaling=fixed) and then click on the "entity" link. If we display this table within the Deployment Side Panel, we'll need to make sure it takes us to the same place. I think we'll probably need to keep the state to determine when to show the External IP Details Page within a common ancestor of the Deployment Side Panel too. That might also mean we'll have to do a lot of prop drilling. Something to think about.

Reply:
Yeah I think ideally we would use routing and not state for determining what to show in this side panel. Thats going to be difficult because routing isn't currently setup as is but I'm hoping it's feasible in the amount of time we have left. I was planning on trying to tackle that early next week.

Reply:
I could definitely see redux and context being useful in network graph but that could be quite the overhaul at this point

Reply:
Just a thought. Whatever is most appropriate for the situation, let's eventually go for that.